### ISSUE TITLE
NoBackend.org seems to assume a world of Web Browsers?

### ISSUE THREAD

**Issue by cconstab** [noBackend/nobackend.org#39]
I submitted a PR to include The @ Company in your list. But cannot implement an example as the challenge set assumes the world of the web browser.
 But the world we live in now is much more about Apps and how Apps communicate without a backend. Web browsers have no safe place to put cryptographic keys and web servers mean you have to trust the admin of the server. Neither seem like reasonable things these days.

Could the challenge be re cast to not be only the browser world ?

**Comment by gr2m** [comment]
The idea for noBackend is that you can use APIs looking native to front end developers that abstract away the use of a backend. So there is a backend, you just don't need to think about it. It seems that the @ Company is focusing on encrypted P2P apps which is out of scope for what noBackend is standing for.

But also not that I coined the term in 2013 (I think). It's more of an artifact representing an idea for it's time, it's not something I actively maintain. I should probably archive the repository

**Comment by cconstab** [comment]
TLDR;

 We love the term so much we built a platform with no backend.. Can we use it and quote you?

TL :-)
@gr2m thanks for the reply. Yes all @platform apps are end to end encrypted (unless its public data). But we also implement is such a way that the backend is not exposed to the developer. A developer using our SDK, just put/get/delete to a seemingly local key-value pair database with one important additional piece of information, which other @sign do they want to allow access to this key-value.

To the frontend developer this means no worry of setting up or managing "users" or infrastructure, but as the people using the app create their own encryption keys with which all the data is end to end encrypted the developer and the infra provider cannot see the data. Only the sending and permitted @signs can see the data.

Each @sign has its own "backend" microservice which can be provided but us or others or run your own, with dess. https://github.com/atsign-foundation/dess

We currently support OSX/Linux/Windows/Android/IoS but have not found a safe way to keep keys in the browser world.

Happy discuss on a VC if that would be interesting for you ? Many of our concepts are very old others are a little more mind bending...
